src/TextUI/Configuration/Cli/Builder.php

```diff
@@ -803,12 +804,24 @@ public function fromParameters(array $parameters): Configuration
                     break;
 
                 case '--log-events-text':
-                    $logEventsText = $option[1];
+                    $path     = $option[1];
+                    $realPath = realpath($path);
+
+                    if (!$realPath) {
+                        throw new Exception("Specified path: {$path} can't be resolved");
```

Comment:
Please avoid using contractions such as `can't`. In this case, I would use `The path "..." specified for the --log-events-text option could not be resolved` as the message.